src/common/include/sirf/common/csirf.h

```diff
@@ -43,6 +43,14 @@ void* cSIRF_axpby(const PTR_FLOAT ptr_a, const void* ptr_x,
 	const PTR_FLOAT ptr_b, const void* ptr_y);
 void* cSIRF_axpbyAlt(const PTR_FLOAT ptr_a, const void* ptr_x,
 	const PTR_FLOAT ptr_b, const void* ptr_y, void* ptr_z);
+void* cSIRF_xapyb_ss(const void* ptr_x, const PTR_FLOAT ptr_a,
+	const void* ptr_y, const PTR_FLOAT ptr_b);
+void* cSIRF_xapyb_ss_Alt(const void* ptr_x, const PTR_FLOAT ptr_a,
+	const void* ptr_y, const PTR_FLOAT ptr_b, void* ptr_z);
+void* cSIRF_xapyb_vv(const void* ptr_x, const void*  ptr_a,
+	const void* ptr_y, const void* ptr_b);
+void* cSIRF_xapyb_vv_Alt(const void* ptr_x, const void*  ptr_a,
+	const void* ptr_y, const void* ptr_b, void* ptr_z);	
```

Comment:
see the above suggestion to replace these four functions with two functions with an extra `const char* args` parameter